templates/store/search.html

```diff
@@ -72,8 +72,12 @@ <h3>All {{ category_display|lower }} snaps</h3>
         {% endif %}
     {% else %}
       <div class="row">
-        <h2>We couldn't find anything for your search <div class="no-search-query">{{query}}.</div></h2>
-        <h4>Try using some different, or more general keywords.</h4>
+        {% if query %}
+          <h2>We couldn't find anything for your search <div class="no-search-query">{{query}}.</div></h2>
+          <h4>Try using some different, or more general keywords.</h4>
+        {% else %}
+          <h2>We couldn't find anything in the category <div class="no-search-query">{{category_display}}.</div></h2>
```

Comment:
Shouldn't we say that category doesn't exist? Or we don't know that, and we only know it doesn't return any results?

Reply:
it's the same for me. What would you prefer?

Reply:
"We couldn't find the category 'name'." ?